package.json

```diff
@@ -123,6 +128,21 @@
       "maxSize": "120 kB"
     }
   ],
+  "nyc": {
+    "report-dir": "coverage/unit",
+    "reporter": ["text", "json", "lcov"],
+    "all": true,
+    "branches": 95,
+    "functions": 90,
+    "lines": 95,
+    "statements": 95,
+    "watermarks": {
+      "branches": [90, 95],
+      "functions": [90, 90],
+      "lines": [90, 95],
+      "statements": [85, 95]
+    }
```

Comment:
these values are all kinda arbitrary for now

the purpose of this is to ensure that we get red (and yellow) colouring in the report for individual files